compiler/src/codegen/compcore.re

```diff
@@ -3323,14 +3322,15 @@ let compile_imports = (wasm_mod, env, {imports}) => {
     | MImportWasm => Ident.name(name)
     | MImportGrain => "GRAIN$MODULE$" ++ Ident.name(name);
 
-  let compile_import_name = name =>
-    fun
```

Comment:
had to double-take on reading the `fun` and then working backwards on this refactor. `fun` is nice, but not clear for my brain during reviews. Nice cleanup! 😄